frontend/src/metabase/actions/components/ActionViz/ActionParameterMapping.tsx

```diff
@@ -68,7 +68,7 @@ export const ActionParameterMappingForm = ({
         setParameterMapping(
           dashboardParameterId,
           dashcard.id,
-          undefined, // this is irrelevant for action parameters
+          -1, // this is irrelevant for action parameters
```

Comment:
I am not sure if that's entirely safe for the FE, but for the BE it [should be](https://metaboat.slack.com/archives/C05MPF0TM3L/p1708005245412669?thread_ts=1708004479.233609&cid=C05MPF0TM3L).

Reply:
Should be safe, as we only genereate [negative IDs for `dashcards`](https://github.com/metabase/metabase/blob/792a4ed7c627a2018de14f149f618dca53866833/frontend/src/metabase/dashboard/utils.ts#L329) and for new queries we don't add IDs as far as I can tell.